x-pack/test/apm_api_integration/basic/tests/service_overview/instances.ts

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import url from 'url';
+import { pick, sortBy } from 'lodash';
+import { isFiniteNumber } from '../../../../../plugins/apm/common/utils/is_finite_number';
+import { APIReturnType } from '../../../../../plugins/apm/public/services/rest/createCallApmApi';
+import { FtrProviderContext } from '../../../../common/ftr_provider_context';
+import archives from '../../../common/archives_metadata';
+
+export default function ApiTest({ getService }: FtrProviderContext) {
+  const supertest = getService('supertest');
+  const esArchiver = getService('esArchiver');
+
+  const archiveName = 'apm_8.0.0';
+  const { start, end } = archives[archiveName];
+
+  interface Response {
+    status: number;
+    body: APIReturnType<'GET /api/apm/services/{serviceName}/service_overview_instances'>;
```

Comment:
Were you not the slightest tempted to write a client when you did this? :p

Reply:
I am 😬 but I would like someone else to pick that up.